app/jinja_filters.py

```diff
@@ -217,8 +217,8 @@ def should_wrap_with_fieldset_processor():
 
 
 @blueprint.app_template_filter()
-def get_width_class_for_number(answer):
-    allowable_widths = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 20]
+def get_width_for_number(answer):
+    allowable_widths = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, 20, 30, 40, 50]
```

Comment:
Was this documented somewhere in the DS?.

Reply:
A lot of work has gone on to improve the docs recently so maybe it wasn't when this was written but yeah it is documented [here](https://ons-design-system.netlify.app/components/input/) now